examples/enc-search-sample.rs

```diff
@@ -0,0 +1,421 @@
+// =============================================================================
```

Comment:
Do we have a stable link to the pattern that this sample demonstrates?

I think it would be good to try to give a very abbreviated description of what this example is illustrating. 
This sentence from the pattern seems like a good start.
> The following implementation patterns illustrate how you can use the IronCore SDKs in your application to do short substring searches of data that is end-to-end encrypted.

Reply:
Done